### PR TITLE
Add high capacity tanks to cargo + some little changes

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

- High capacity tanks was added to the cargo
- Added empty high capacity tanks
- Some cleanup and rearrangment
- All tanks now cost 500 spesos

## Why / Balance

Cool

## Technical details

no

## Media

no

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

no

**Changelog**

Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: High capacity water and welding fuel  tanks was added to the cargo request console.

### CODE REVIEW

File: Resources/Prototypes/Catalog/Cargo/cargo_materials.yml [hunk 1]
```diff
@@ -97,3 +97,23 @@
...
+
+- type: cargoProduct
+  id: MaterialHighCapacityWaterTank
+  icon:
+    sprite: Structures/Storage/tanks.rsi
+    state: highwatertank
+  product: WaterTankHighCapacityFull
+  cost: 5000
+  category: Materials
+  group: market
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Cargo/cargo_materials.yml:119]: Why would you buy regular vs high capacity.

**Comment by lzk228** [comment]
removed gonflicts

**Comment by lzk228** [comment]
so what with the discussion

**Comment by UbaserB** [comment]
i would say lock it as a tank upgrade behind science and add a new tech that handles transportation of tank contents.

**Comment by metalgearsloth** [comment]
omega conflicts + review never addressed.

**Pull request closed without merging**